recipes/bustools/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "0.39.2" %}
+
+package:
+    name: bustools
+    version: {{ version }}
+
+source:
+    url: https://github.com/BUStools/bustools/archive/v{{ version }}.tar.gz
+    sha256: f42184fdd20309b1f77d09e97e8f9005b81ab030eb924ce69f6c62cedd50ce57
+
+build:
+    number: 0
+    string: BUStools
+
+requirements:
+    build:
+        - {{ compiler('cxx') }}
+        - cmake
+    run:
+        - hdf5
+        - zlib
+    host:
```

Comment:
both runtime deps do have run_exports defined and can be removed.

Reply:
Just removed that as well